Misc/clientorder_name.csv

```diff
@@ -134,7 +134,7 @@ Notes_002#0,"""This order cannot be accepted<br>until $(0)."""
 Notes_003#0,"""This order can be completed<br>any number of times."""
 Notes_004#0,"""This order can be accepted until 23:59<br>JST."""
 Notes_005#0,"""This order has already been completed<br>and is no longer available."""
-Notes_006#0,"""This order is available until<br><utcdow>Wed 2:00</utcdow>."""
+Notes_006#0,"""This order is available until<br><utcdow>Wed 2:00</utcdow>"""
```

Comment:
Why have you made this change?

Reply:
![image](https://user-images.githubusercontent.com/17720989/110261958-0893bd80-7faa-11eb-8b41-e49d2874929d.png)